setup/src/setup.cpp

```diff
@@ -30,21 +31,39 @@ void copyVideos(std::wstring videoDir, std::string outputDir) {
  *
  * The code is very non-portable.
  */
-int main() {
+int main( int argc, char* argv[] ) {
 
-    // Read original game directory from registry
     std::wstring gameDir;
-    try {
-        gameDir = Registry::RegGetString(
+    bool findDirectoryFromRegistry = true;
+    for ( int count{ 1 }; count < argc; ++count ) {
+        if ( "-h" == argv[count] ) {
+            std::cout << "-d {directory}: loads all files from {directory}" << '\n';
+            return 0;
+        }
+        if ( 0 == strcmp( "-d", argv[count] ) and count + 1 <= argc ) {
```

Comment:
I've used `&&` elsewhere - in fact I didn't even know `and` was a valid keyword until now!